LevelComponents/Room.cpp

```diff
@@ -1001,17 +992,26 @@ namespace LevelComponents
     }
 
     /// <summary>
-    /// Set Layer data pointer into the Room header struct in sake of saving changes.
+    /// Set a data pointer in RoomHeader struct used for changes based on saving and creating new Rooms.
     /// </summary>
-    void Room::SetLayerDataPtr(int LayerNum, int dataPtr)
+    void Room::SetRoomHeaderDataPtr(int pointerId, int dataPtr)
     {
         // this can be used to set entity set data pointers too
-        if(LayerNum & 0xFFFFFFFC)
+        switch(pointerId)
```

Comment:
Don't change this to a switch, use the if statement and change the constant to 0xFFFFFFF8

Reply:
Done.